mrp_multi_level_estimate/wizards/mrp_multi_level.py

```diff
@@ -122,8 +122,11 @@ def _prepare_mrp_move_data_from_stock_move(
         )
         if direction == "out":
             mrp_date = res.get("mrp_date")
-            if self._exclude_considering_estimate_demand_and_other_sources_strat(
-                product_mrp_area, mrp_date
+            if (
+                self._exclude_considering_estimate_demand_and_other_sources_strat(
+                    product_mrp_area, mrp_date
+                )
+                and not res["production_id"]
```

Comment:
I think it should be raw_material_production_id to refer to indirect demand. Please, add a test case too.